media-sound/ncmpcpp/ncmpcpp-0.9.2_p20240724.ebuild

```diff
@@ -0,0 +1,74 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
+
+NCMPCPP_COMMIT="68daf44032784a5b5b74781a9bf3826053eff4a2"
+
+DESCRIPTION="Featureful ncurses based MPD client inspired by ncmpc"
+HOMEPAGE="
+	https://rybczak.net/ncmpcpp/
+	https://github.com/ncmpcpp/ncmpcpp/
+"
+SRC_URI="https://github.com/ncmpcpp/ncmpcpp/archive/${NCMPCPP_COMMIT}.tar.gz -> ${P}.gh.tar.gz"
+S="${WORKDIR}/${PN}-${NCMPCPP_COMMIT}"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE="clock -lto outputs taglib visualizer"
```

Comment:
Now you have added IUSE="lto", but the corresponding option in `./configure` only triggers the `-flto` flag. Correct me if I'm wrong, but we already have `C[XX]FLAGS` for this, so this USE flag should really be necessary.

Reply:
Correct, see 4b40de2810608796d824ccb4baed84e1a5599c79 and 783d5b43d785767b21e197a3570c857329d59f0c

Reply:
No. You are literally reverting 783d5b43d785767b21e197a3570c857329d59f0c

TLDR we try to avoid using use flags to control setting `*FLAGS` as much as possible. If the `lto` has **no other function** in the ebuild, than adding `-flto` to your `*FLAGS`, then it's invalid use of use flag.